api/client/plugin/list.go

```diff
@@ -33,11 +33,11 @@ func runList(dockerCli *client.DockerCli) error {
 	}
 
 	w := tabwriter.NewWriter(dockerCli.Out(), 20, 1, 3, ' ', 0)
-	fmt.Fprintf(w, "NAME \tTAG \tACTIVE")
+	fmt.Fprintf(w, "NAME \tTAG \tDESCRIPTION\tACTIVE")
 	fmt.Fprintf(w, "\n")
 
 	for _, p := range plugins {
-		fmt.Fprintf(w, "%s\t%s\t%v\n", p.Name, p.Tag, p.Active)
+		fmt.Fprintf(w, "%s\t%s\t%s\t%v\n", p.Name, p.Tag, p.Manifest.Description, p.Active)
```

Comment:
This is the code used in `docker search`; [api/client/image/search.go#L111-L114](https://github.com/docker/docker/blob/a502158d8a9b3a3949eb0bc0b087138ef65efabb/api/client/image/search.go#L111-L114)

perhaps we should enhance the `stringutils.Truncate` to take an extra parameter for an ellipsis